### DISCUSSION TITLE
Big problem

### DISCUSSION THREAD

**Discussion by Sonkinator1991** [Open-Shell/Open-Shell-Menu#1705] [General]
### Describe the bug

Hi, I tried to uninstall open shell, but in the process, my taskbar disappeared and my screen shows just gray and 3 white rectangles. nothing else. I cant add a screenshot, because this was on my laptop, which runs windows 11. It's 4 years old. If you can give me an answer, please do

### Area of issue

Installation/Other

### To reproduce

idk if u can

### Expected behavior

i expected to have open shell uninstalled

### Open-Shell version

latest

### Windows version

windows 11

**Comment by ge0rdi** [comment]
What does happen when you reboot your machine?

**Reply by Sonkinator1991** [reply]
how would I? my taskbars gone, is there a key combination i should use?

**Reply by ge0rdi** [reply]
You can start `Task Manager` by pressing `Ctrl + Shift + Esc`.
Then you can try to start `Explorer` (that is most likely not running) via `File` -> `Run new task` (just type `explorer.exe` there).

If that will do nothing, then most likely Explorer has some trouble with starting. It may be caused by some references to `Open-Shell` that weren't removed during uninstall (?).
You can then try manual uninstall of `Open-Shell`:

- start your web browser (via Task Manager as described above)
- download our [Utility](https://github.com/Open-Shell/Open-Shell-Menu/releases/download/v4.4.191/Utility.exe) and run it
- select `Remove Open-Shell`
- follow instructions there to remove `Open-Shell` manually

**Reply by Sonkinator1991** [reply]
swag i'll see what happen

**Reply by Sonkinator1991** [reply]
ITS WORKING, ITS WORKING!

**Reply by Sonkinator1991** [reply]
IT WORKED! You are a lifesaver. I was so worried that my my laptop would kick the bucket, but you knew what to do! I am SO grateful that you could help me with this.

**Reply by ge0rdi** [reply]
Could you possibly share what step helped?

**Comment by Sonkinator1991** [comment]
Opening explorer brought my wallpaper and (If im not mistaken) Taskbar back

**Reply by ge0rdi** [reply]
Oki, thanks.

Explorer is supposed to be restarted once installer finishes (if there were some `Open-Shell` binaries still in use).
Apparently it didn't restart correctly in your case. Hard to tell why ...
